grpc-interface-support/src/cpp.py

```diff
@@ -497,39 +491,48 @@ def __install_protoc_via_conan(self, conan_build_dir: str) -> None:
                     match = pattern.match(line)
                     if match is not None:
                         deps_results.append(match.group(1))
+                        deps_patterns.remove(pattern)
 
-        tmpdir = tempfile.mkdtemp()
-        tooling_conanfile_path = os.path.join(tmpdir, "conanfile.txt")
+        tooling_conanfile_path = os.path.join(conan_build_dir, "conanfile.txt")
         with open(
             tooling_conanfile_path, mode="w", encoding="utf-8"
         ) as tooling_conanfile:
-            tooling_conanfile.write("[requires]\n" + "\n".join(deps_results))
+            tooling_conanfile.write("[requires]\n" + "\n".join(deps_results) + "\n\n")
+            tooling_conanfile.write(
+                "[tool_requires]\ngrpc/<host_version>\nprotobuf/<host_version>\n\n"
+            )
+            tooling_conanfile.write("[generators]\nCMakeDeps\nCMakeToolchain\n")
 
         subprocess.check_call(
             [
                 "conan",
                 "install",
-                "-pr:h",
+                "-pr:a",
                 CONAN_PROFILE_NAME,
+                "-s:a=compiler.libcxx=libstdc++11",
```

Comment:
is this not part of the profile file?

Reply:
I also don't know exactly, just know that this is part of our other profiles and a line above a profile is used. maybe we can add it to this profile instead of the sourcecode?

Reply:
Removed